from0k2bp.tex

```diff
@@ -1159,33 +1179,42 @@
 that $z = \mathbf{x}\cdot \mathbf{y}$. So our Extractor is tasked with extracting those vectors and must
 be able to verify that the dot product equation holds.
 
-First let's note the transcript: it's $\left((A_d, B_d, C_1, C_0), e, (\textbf{f}_x, \textbf{f}_y, r_x, s_y, t_z)\right)$. As a reminder: the 'commitment'
-step of the Sigma protocol involves sending those 4 curve points (each
-of which are themselves Pedersen commitments), then the Verifier sends
-back a single scalar challenge, then the 'response' step involves the
-Prover sending two blinded vectors (f), along with random scalars to
-make the Pedersen commitments verify correctly, and finally the $t_z$ to allow
-the completed verification of the inner product.
+First let's note the transcript: 
+\begin{align*}
+\left((A_d, B_d, C_1, C_0), e, (\textbf{f}_x, \textbf{f}_y, r_x, s_y, t_z)\right)
+\end{align*}
+As a reminder: the 'commitment' step of the Sigma protocol involves sending:
+
+\begin{itemize}
+\item those 4 curve points -- each of which are Pedersen commitments themselves,
+\item then the Verifier sends back a single scalar challenge (i.e. $e$), 
+\item then the 'response' step involves the Prover sending two blinded vectors (f), along with random scalars to
+make the Pedersen commitments verify correctly, and finally
+\item $t_z$ to allow the completed verification of the inner product.
+\end{itemize}
 
-What can we do if the Extractor gets the Prover to output a second
-accepting transcript with the same commitment, say $\left((A_d, B_d, C_1, C_0), e', (\textbf{f}_x', \textbf{f}_y', r_x', s_y', t_z')\right)$?
+What can we do if the Extractor gets the Prover to output
+a second accepting transcript with the same commitment, 
+say $\left((A_d, B_d, C_1, C_0), e', (\textbf{f}_x', \textbf{f}_y', r_x', s_y', t_z')\right)$?
 
 Let's isolate our consideration to the \textbf{x}-part only: we have two
 validating checks, one from each transcript:
 \begin{align*}
 & eC_x + A_d = r_xH + \mathbf{f}_x\mathbf{G} \\
-& e' C_x + A_d = r_x' H + \mathbf{f}_x' \mathbf{G} \\
+& e' C_x + A_d = r_x' H + \mathbf{f}_x' \mathbf{G}
 \end{align*}
-Subtracting the second equation from the first and multiplying both
+Subtracting the second equation from the first one and multiplying both
 sides by $\eta = (e-e')^{-1}$ gives:
 \[C_x = \eta (r_x - r_x') H + \eta (\mathbf{f}_x - \mathbf{f}_x' ) \mathbf{G}\]
 and we note by the same logic as for the set-of-vectors case, that this,
 by the binding property of the Pedersen commitment, proves that
 \[\mathbf{x} = \eta (\mathbf{f}_x - \mathbf{f}_x')\]
-We can also now extract the opening of the commitment $A_d$: it's just $\mathbf{d}_x = \mathbf{f}_x - e\mathbf{x}$, for
-example.
-
-\ldots{} and the same procedure can of course extract $\mathbf{y}, \mathbf{d}_y$.
+We can also now extract the opening of $A_d$ commitment\footnote{See definition \eqref{inner_product_proof_AB_d_def:1}}: 
```

Comment:
"opening of the commitment $A_d$" is better than "opening of $A_d$ commitment", change not needed.

Reply:
Fixed